src/lib/initEnv.js

```diff
@@ -15,6 +15,7 @@ window.RUNNING_IN_POPUP =
 window.POPUP_TYPE = url.searchParams.get('type') || null;
 window.IS_EXTENSION_BACKGROUND =
   process.env.IS_EXTENSION && window.location.href.endsWith('_generated_background_page.html');
+window.IS_DESKTOP = document.documentElement.clientWidth > 480;
```

Comment:
this value can become outdated if the user changes the window size, I'm proposing to avoid this, at least it may complicate testing